INSTALL.md

```diff
@@ -75,14 +110,18 @@ echo 'export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/9.5/bin/"'
 source ~/.bash_profile
 ```
 
-### Redis-server installation on Mac OS X
+### Redis-server installation on Mac OS X or Linux
```

Comment:
@aliu104 I would suggest breaking this up into two sections, one for MacOSX and one for Linux just as you did above. The reason for this is home-brew is not a Linux thing (unless things have recently changed) and apt-get is very specifically an _Ubuntu_ thing (not even an Linux thing).

Reply:
@charles-cowart I wasn't too familiar with home-brew, but after searching it up, it seems that home-brew also works for Linux now: https://brew.sh/. Should I still break up these sections?

Reply:
@aliu104 Ooh that is interesting news. Yes, we should still break it up into sections, since the second method is specifically for Ubuntu. IMHO, The title of the first section should reflect on Homebrew and its cross-platform nature specifically. "Redis-server installation using Homebrew (Mac OS X, Linux)" would be my input.